deploy/nns.go

```diff
@@ -190,7 +191,7 @@ func lookupNNSDomainRecord(inv *invoker.Invoker, nnsContract util.Uint160, domai
 	item, err := unwrap.Item(inv.Call(nnsContract, methodNNSResolve, domainName, int64(nns.TXT)))
 	if err != nil {
 		var except unwrap.Exception
-		if errors.As(err, &except) && string(except) == "token not found" {
+		if errors.As(err, &except) && strings.Contains(string(except), "token not found") {
```

Comment:
became less laconic at this scope to me. Strange to see `invocation failed: at instruction 4100 (THROW): unhandled exception: \"token not found\""` as `unwrap.Exception` when (Go) contract just makes `panic("token not found")`. Is it possible to improve somehow to keep previous code?

Reply:
It's not, RPC server returns `at instruction 4100 (THROW): unhandled exception: "token not found"`. Exception here means `exception` field in the RPC protocol which is a broader thing than contract's exception (ABORT will produce a similar one, for example, but it's not an exception at all).